stages/org.osbuild.rpm

```diff
@@ -111,6 +111,10 @@ SCHEMA_2 = """
       "description": "Prevent dracut from running",
       "type": "boolean"
     },
+    "first_boot": {
+      "description": "Trigger the FirstBoot condition of systemd on boot.",
+      "type": "boolean"
+    },
```

Comment:
I wonder if we not rather should describe what we are doing to `machine_id`, instead of the effect this will cause.